db/schema.rb

```diff
@@ -203,6 +203,12 @@
     t.decimal  "discount",        :precision => 10, :scale => 2, :default => 0.0
   end
 
+  create_table "debit_direct_files", :force => true do |t|
```

Comment:
Good point! I really missed these changes... Could you have a look again, @huerlisi?